otx/algorithms/common/adapters/mmcv/runner.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, *args, **kwargs):
         self.should_stop = False
         _, world_size = get_dist_info()
         self.distributed = world_size > 1
-        self.data_loader = None
```

Comment:
Is it correct that self.data_loader removed?

Reply:
Yeah,, it was removed in commits in recent develop. I checked it..